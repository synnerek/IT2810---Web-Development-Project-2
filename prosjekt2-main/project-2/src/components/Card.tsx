import React from "react";

type CardProps = {
    authorName: string;
    title: string;
    comittedDate: string;
}

export default function Card({authorName,title,comittedDate}: CardProps){
    return( 
    <div className="commitsBox">
        <p className="text-xl">{title}</p>
        <p>Author: {authorName}</p>
        <p>Comitted at: {comittedDate.substring(0,10)}</p>


    </div>)
}