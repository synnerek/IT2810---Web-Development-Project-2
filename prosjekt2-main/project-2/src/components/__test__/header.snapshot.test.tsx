import React, {useState} from 'react';
import renderer from 'react-test-renderer';
import Card from '../Card';


it('renders correctly', () => {
  const tree = renderer
    .create(<Card authorName={''} title={''} comittedDate={""} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

