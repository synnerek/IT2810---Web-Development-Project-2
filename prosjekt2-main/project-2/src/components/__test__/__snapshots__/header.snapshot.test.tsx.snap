// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="commitsBox"
>
  <p
    className="text-xl"
  />
  <p>
    Author: 
  </p>
  <p>
    Comitted at: 
  </p>
</div>
`;
